FirebaseAppCheck/Sources/Core/APIService/FIRAppCheckAPIService.h

```diff
@@ -29,7 +29,7 @@ NS_ASSUME_NONNULL_BEGIN
 - (FBLPromise<GULURLSessionDataResponse *> *)
     sendRequestWithURL:(NSURL *)requestURL
             HTTPMethod:(NSString *)HTTPMethod
-                  body:(NSData *)body
+                  body:(nullable NSData *)body
```

Comment:
Added `nullable` attribute because no body is sent when requesting random challenge.